ajax/libs/rita/package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "rita",
+  "version": "1.1.40",
+  "author": "Daniel C. Howe <daniel@rednoise.org>",
+  "description": "RiTa: A toolkit for natural language and computational literature",
+  "homepage": "http://rednoise.org/rita",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/dhowe/RiTa.git"
+  },
+  "keywords": [
+    "nlp",
+    "natural language",
+    "language",
+    "writing",
+    "generative"
+  ],
+  "license": "GPL-3.0",
+  "tonicExampleFilename": "tonic.js",
+  "optionalDependencies": {
+    "yamljs": "^0.2.3"
+  },
+  "npmName": "rita",
+  "npmFileMap": [
+    {
+      "basePath": "/dist",
```

Comment:
There is no `dist` folder in npm package. I think we should use git auto-update here but I'm not sure. @PeterDaveHello Please give us some suggestions, thank you.